.changeset/loud-sheep-lay.md

```diff
@@ -0,0 +1,6 @@
+---
+'@marigold/components': minor
+'@marigold/docs': minor
+---
+
+Migrate Select component to RAC
```

Comment:
This breaking change info should also be added here: https://github.com/marigold-ui/marigold/pull/3546/files#diff-7617ce6947d6789d501bb7df466ddca07d7d0aa5264a4c3f43f9cdbbed05f22b